src/Responses/ListProfilesResponse.php

```diff
@@ -22,7 +22,7 @@ class ListProfilesResponse extends AbstractResponse
     /**
      * @var array
      */
-    protected $profiles = [];
+    protected $profiles = array();
```

Comment:
short array syntax is introduced in php 5.4 but according to `composer.json` minimum requirement is php 5.3.3